Source/WebCore/Modules/indexeddb/server/UniqueIDBDatabase.h

```diff
@@ -112,6 +112,8 @@ class UniqueIDBDatabase : public CanMakeWeakPtr<UniqueIDBDatabase> {
     void commitTransaction(UniqueIDBDatabaseTransaction&, uint64_t handledRequestResultsCount, ErrorCallback&&, SpaceCheckResult = SpaceCheckResult::Unknown);
     void abortTransaction(UniqueIDBDatabaseTransaction&, ErrorCallback&&, SpaceCheckResult = SpaceCheckResult::Unknown);
 
+    Vector<String> storedBlobPathsForTransaction(UniqueIDBDatabaseTransaction&);
```

Comment:
Where is this used? What's the purpose of this change?

Reply:
It's vestigial. I will remove it.